database/migrations/2017_11_08_123942_labels_display_company_name.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+use Illuminate\Support\Facades\Schema;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Database\Migrations\Migration;
+
+class LabelsDisplayCompanyName extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::table('settings', function (Blueprint $table) {
+            $table->tinyInteger('labels_display_company_name')->default(0);
```

Comment:
Probably better to make this is `boolean`?

`$table->boolean('labels_display_company_name')->default(0);`